### PR TITLE
Add release notes for `v0.3.0`

### PR DESCRIPTION
**Pull request by afilini**

### CODE REVIEW

File: content/blog/2021/release-0.3.0.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+title: "Release v0.3.0"
+description: "Announcing the v0.3.0 release of BDK"
+author: "Alekos Filini"
+date: "2021-01-20"
+tags: ["rust", "release"]
+hidden: true
+draft: false
+---
+
+A new release of BDK is out: the `v0.3.0` is a relatively small update compared to `v0.2.0`, but it still brings some nice APIs improvements and general bugfixes.
```
Comment (RCasatta) [inline content/blog/2021/release-0.3.0.md]:
```suggestion
A new release of BDK is out: the [`v0.3.0`] is a relatively small update compared to `v0.2.0`, but it still brings some nice APIs improvements and general bugfixes.
```

Maybe add a link to crates?

File: content/blog/2021/release-0.3.0.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+- [@tcharding][@tcharding] - Tobin C. Harding
+
+[changelog]: https://github.com/bitcoindevkit/bdk/blob/75669049268bbc294564f8c6e0528e07a546258f/CHANGELOG.md#v030---v020
+[gh_diff]: https://github.com/bitcoindevkit/bdk/compare/v0.2.0...v0.3.0
+[bdk-cli]: https://github.com/bitcoindevkit/bdk-cli
+
+[`descriptor!()`]: https://docs.rs/bdk/0.3.0/bdk/macro.descriptor.html
+[`descriptor::error::Error`]: https://docs.rs/bdk/0.3.0/bdk/descriptor/error/enum.Error.html
+
```
Comment (RCasatta) [inline content/blog/2021/release-0.3.0.md:75]:
```suggestion
[`v0.3.0`]: https://crates.io/crates/bdk/0.3.0
```

**Comment by notmandatory** [comment]
ACK d374aec, I also double checked the `bdk` changelog and links and text look good there too.

File: content/blog/2021/release-0.3.0.md (hunk 2 above)
Comment (notmandatory) [reply content/blog/2021/release-0.3.0.md:75]: Good idea :+1:

**Pull request merged**